src/Forms/ClientForm.php

```diff
@@ -52,7 +54,7 @@ class ClientForm extends Form
     /**
      * @throws Exception
      */
-    public function __construct(private readonly ModuleConfig $moduleConfig)
+    public function __construct(private readonly ModuleConfig $moduleConfig, protected CsrfProtection $csrProtection)
```

Comment:
The variable name is missing the `f`.  `$csrfProtection`.  I think there are a couple places with the typo

Reply:
done